ibis/backends/tests/tpc/ds/test_queries.py

```diff
@@ -1353,6 +1352,7 @@ def test_26(catalog_sales, customer_demographics, date_dim, item, promotion):
 
 
 @tpc_test("ds")
+@pytest.mark.notyet(["datafusion"], reason="Failed to plan")
```

Comment:
This is a new bug, I'll file an upstream issue when I get a chance to narrow down a reproducer.